SWLOR.Game.Server/Service/Spawn.cs

```diff
@@ -473,6 +473,51 @@ private static void ProcessDespawnAreas()
             }
         }
 
+        /// <summary>
+        /// On each module heartbeat, iterate over the list of areas and trigger
+        /// respawn on expired Spawns.
+        /// </summary>
+        private static void ProcessRespawnAreas()
+        {
+            var now = DateTime.UtcNow;
+
+            foreach(var _area in _activeSpawnsByArea)
```

Comment:
This could be thousands of iterations every six seconds. This is going to be too heavy. We need to come up with another approach.